src/Components/SearchBar/index.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Grid, Paper, TextField, Button } from "material-ui";
+
+const styles = {
```

Comment:
Const styles should be imported from the component folder, from a file named styles.css.